storage/spec/files_spec.rb

```diff
@@ -182,6 +185,94 @@ def capture &block
     end
   end
 
+  it "can download a file from a bucket using requester pays" do
+    begin
+      @bucket.requester_pays = true
+      expect(@storage.bucket(@bucket_name).requester_pays).to be true
+
+      delete_file "file.txt"
+      expect(@bucket.file "file.txt").to be nil
+
+      upload @local_file_path, "file.txt"
+
+      local_file = Tempfile.new "cloud-storage-tests"
+      expect(File.size local_file.path).to eq 0
+
+      @bucket.policy do |policy|
+        policy.roles["roles/storage.objectViewer"] = []
+        policy.add "roles/storage.objectViewer", "allUsers"
+      end
+
+      expect(Google::Cloud::Storage).to receive(:new).
+                                        with(project: @project_id_secondary).
+                                        and_return @storage_secondary
+
+      expect {
+        download_file_requester_pays project_id:  @project_id_secondary,
+                                     bucket_name: @bucket_name,
+                                     local_path:  local_file.path,
+                                     file_name:   "file.txt"
+      }.to output(
+        "Downloaded file.txt using billing project #{@project_id_secondary}\n"
+      ).to_stdout
+
+      expect(File.size local_file.path).to be > 0
+      expect(File.read local_file.path).to eq(
+        "Content of test file.txt\n"
+      )
+
+      @bucket.policy do |policy|
```

Comment:
Is this cleanup code? 

If storage specs need the bucket policy to be set in a certain way, let's override the bucket policy in the before block 

Cleanup code should be in the after block - because the test could blow up on any line! If it fails, the cleanup code will never run 😄